src/video/windows/SDL_windowsevents.c

```diff
@@ -1247,6 +1247,45 @@ void SDL_SetWindowsMessageHook(SDL_WindowsMessageHook callback, void *userdata)
     g_WindowsMessageHookData = userdata;
 }
 
+int
+WIN_WaitEventTimeout(_THIS, int timeout)
+{
+    MSG msg;
+    if (g_WindowsEnableMessageLoop) {
```

Comment:
If `g_WindowsEnableMessageLoop` is false and we always return 0 without waiting, is the caller going to handle that properly?

Reply:
This should be fixed by the same commit of above: https://github.com/franko/SDL-1/commit/a3820fd24eb4a85e179fcaf90e5b477c08dd3a82.

Now if g_WindowsEnableMessageLoop is false the SDL_WaitEventTimeout immediately return 0. Should it wait the timeout to elapse before returning ? Maybe yes because the previous implementation behaves this way.